tests/core/test_ensemble.py

```diff
@@ -11,6 +11,13 @@
 from rasa.core.trackers import DialogueStateTracker
 from rasa.core.events import UserUttered
 
+from tests.core.utilities import user_uttered
```

Comment:
let's not import the function directly (https://www.notion.so/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e)

Reply:
fixed =)